ballerina/build.gradle

```
@@ -456,12 +456,12 @@ task packageDistLinux(type: Zip) {
         filter(ReplaceTokens, tokens: [uuid: installerVersion])
     }
     into("${parentDir}/bin") {
-        from "build/target/extracted-distributions/ballerina-command-zip/bin/bal"
+        from "build/target/extracted-distributions/ballerina-command-zip//ballerina-command-${ballerinaCommandVersion}/bin/bal"
```

There is an additional `/`
```suggestion
        from "build/target/extracted-distributions/ballerina-command-zip/ballerina-command-${ballerinaCommandVersion}/bin/bal"
```